webapp/channels/src/components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -169,32 +162,22 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
         );
 
         if (isCloud) {
-        // if all conditions are set for being able to request a cloud trial, then show the cta start cloud trial button
-            if (canRequestCloudFreeTrial) {
-                ctaPrimaryButton = (
-                    <FeatureDiscoveryCloudStartTrialButton
-                        telemetryId={`start_cloud_trial_from_${this.props.featureName}`}
-                        extraClass='btn btn-primary'
+            // if all conditions are set for being able to request a cloud trial, then show the cta start cloud trial button
+            ctaPrimaryButton = (
+                <button
+                    className='btn btn-primary'
+                    data-testid='featureDiscovery_primaryCallToAction'
+                    onClick={() => {
+                        trackEvent(TELEMETRY_CATEGORIES.SELF_HOSTED_ADMIN, 'click_enterprise_contact_sales_feature_discovery');
```

Comment:
This is pre-existing but seems to be labeled in the wrong telemetry category if this is a cloud case?